sdk/python/kfp/compiler/_container_builder.py

```diff
@@ -18,19 +18,31 @@
 import os
 import uuid
 
+SERVICEACCOUNT_NAMESPACE = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
```

Comment:
Is this universal or does it only work in Kubeflow notebooks?

Reply:
it happens in k8s clusters. And there is also checks in the code that defaults to 'kubeflow'